lineman/app/components/move_thread_form/move_thread_form.coffee

```diff
@@ -12,3 +12,13 @@ angular.module('loomioApp').factory 'MoveThreadForm', ->
       flashSuccess: 'move_thread_form.messages.success'
       flashOptions:
         name: -> $scope.discussion.group().name
+
+    $scope.updateTarget = ->
+      $scope.targetGroup = Records.groups.find($scope.discussion.groupId)
+    $scope.updateTarget()
+
+    $scope.moveThread = ->
+      if $scope.discussion.private == true and $scope.targetGroup.privacyIsOpen()
+        $scope.submit() if confirm($filter('translate')('move_thread_form.confirm_change_to_private_thread', groupName: $scope.targetGroup.name))
```

Comment:
Check out `privacy_string_service.coffee` for an example of $translate.instant, which I think is a nicer way to handle this